lib/katana/guardians_ninjas/guardian_ninja.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Katana.GuardiansNinjas.GuardianNinja do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  @foreign_key_type :binary_id
+  schema "guardian_ninja" do
```

Comment:
Join table should be named in their plural form, by convention. `guardians_ninjas`.